profile/BBmri-observation-bodyweight.StructureDefinition.json

```diff
@@ -0,0 +1,42 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "https://fhir.bbmri.de/StructureDefinition/Bbmri-observation-bodyweight",
```

Comment:
Kebab case, starting with upper case char looks not right to me. I would convert that to lower case.